bucket/hourglass.json

```
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://chris.dziemborowicz.com/apps/hourglass",
+    "description": "The simple countdown timer for Windows.",
+    "version": "1.10.0",
+    "license": "MIT",
+    "url": "https://github.com/dziemborowicz/hourglass/releases/download/v1.10.0/HourglassPortable.exe",
+    "hash": "d61981ef85b0b9b6142d30f6bc16f8bfd22efd477914ecd6a6794a5ecfd961b6",
+    "bin": [
+        [
+            "HourglassPortable.exe",
+            "hourglass"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "HourglassPortable.exe",
+            "Hourglass"
+        ]
+    ],
+    "persist": "HourglassPortable.config",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\HourglassPortable.config\")) {",
+        "    New-Item \"$dir\" -Name 'HourglassPortable.config' -ItemType File | Out-Null",
```

- When you are surrounding $dir in quotes it is non sense to not join path and child.
- File is default for new-item since powershell 5

```suggestion
        "    New-Item \"$dir\\HourglassPortable.config\" | Out-Null",
```